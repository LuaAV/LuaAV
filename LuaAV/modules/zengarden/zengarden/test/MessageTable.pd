#N canvas 387 158 460 377 10;
#X obj 53 18 loadbang;
#X obj 122 20 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 167 183 tabwrite array_one;
#X msg 269 112 1;
#X msg 173 129 5;
#X msg 235 112 2;
#X msg 140 129 10;
#X msg 68 102 set array_two;
#X msg 204 112 3;
#X msg 109 129 15;
#X msg 55 129 6;
#X msg 15 121 12;
#X obj 313 165 table array_one;
#X obj 312 201 table array_two;
#X obj 53 47 t b b b b b b b b b b b b;
#X obj 128 291 tabread array_two;
#X obj 53 195 t b b b b b b;
#X msg 153 256 1;
#X msg 123 256 2;
#X obj 128 328 print;
#X msg 82 223 set array_one;
#X msg 62 256 3;
#X connect 0 0 14 0;
#X connect 1 0 14 0;
#X connect 3 0 2 1;
#X connect 4 0 2 0;
#X connect 5 0 2 1;
#X connect 6 0 2 0;
#X connect 7 0 2 0;
#X connect 8 0 2 1;
#X connect 9 0 2 0;
#X connect 10 0 2 0;
#X connect 11 0 2 0;
#X connect 14 0 16 0;
#X connect 14 1 11 0;
#X connect 14 2 5 0;
#X connect 14 3 10 0;
#X connect 14 4 3 0;
#X connect 14 5 7 0;
#X connect 14 6 9 0;
#X connect 14 7 8 0;
#X connect 14 8 6 0;
#X connect 14 9 5 0;
#X connect 14 10 4 0;
#X connect 14 11 3 0;
#X connect 15 0 19 0;
#X connect 16 0 21 0;
#X connect 16 1 18 0;
#X connect 16 2 17 0;
#X connect 16 3 20 0;
#X connect 16 4 18 0;
#X connect 16 5 17 0;
#X connect 17 0 15 0;
#X connect 18 0 15 0;
#X connect 20 0 15 0;
#X connect 21 0 15 0;
