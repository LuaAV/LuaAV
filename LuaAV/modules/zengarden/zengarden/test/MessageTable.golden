[@ 0.000ms] print: 6
[@ 0.000ms] print: 12
[@ 0.000ms] print: 5
[@ 0.000ms] print: 10
[@ 0.000ms] print: 15